instrumentation/opentelemetry_oban/test/opentelemetry_oban_test.exs

```diff
@@ -138,24 +141,26 @@ defmodule OpentelemetryObanTest do
 
     assert_receive {:span,
                     span(
-                      name: "TestJob process",
+                      name: "process TestJob",
                       attributes: attributes,
                       kind: :consumer,
                       status: :undefined
                     )}
 
     assert %{
-             "messaging.destination": "events",
-             "messaging.destination_kind": :queue,
+             "messaging.system": :oban,
+             "messaging.destination.name": "TestJob",
+             "messaging.consumer.group.name": "events",
```

Comment:
So this means we're making the interpretation that for the latest "Semantic conventions for messaging spans"

[Consumer group](https://opentelemetry.io/docs/specs/semconv/messaging/messaging-spans/#consumer-groups) => Oban queue
[Destination](https://opentelemetry.io/docs/specs/semconv/messaging/messaging-spans/#destinations) => Oban Worker

right?